src/components/PasswordStrengthMeter/index.tsx

```diff
@@ -66,8 +67,9 @@ const PasswordStrengthMeter = ({
   userInputs = [],
 }: PasswordStrengthMeterProps): JSX.Element => {
   const [score, setScore] = useState<number>(0)
+  const theme = useTheme()
   const [backgroundColor, setBackgroundColor] = useState<string>(
-    strength[0]?.color || 'green',
+    strength[0]?.color || theme.colors.success.background,
```

Comment:
```suggestion
    strength[0]?.color || theme.colors.success.backgroundStrong,
```

Should be better no ? As default background is really light

Reply:
yes, done